plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -843,11 +895,16 @@ let stm include_ config ir =
     let descr = estring (module_name ^ " STM tests") in
     [%stri
       let _ =
+        let open QCheck in
```

Comment:
Is `QCheck` necessary for more than just `Test.make`? If not, I would rather go with `QCheck.Test.make`.